scripts/test

```diff
@@ -9,6 +9,12 @@ set -x
 
 export CATTLE_TEST_AGENT_IMAGE PHANTOMJS_BIN
 
+if [ -z "${CATTLE_TEST_PARALLEL_THREADS}" ]; then
```

Comment:
Can you use the same paradigm for setting this as we do on lines 6-8. Specifically*:
```
:  ${CATTLE_TEST_PARALLEL_THREADS:=1}
```
(not explicitly tested, but that should work)
* Thanks @cloudnautique